shapely/tests/test_set_operations.py

```diff
@@ -66,6 +73,8 @@ def test_set_operation_array(a, func):
 @pytest.mark.parametrize("func", SET_OPERATIONS)
 @pytest.mark.parametrize("grid_size", [0, 1])
 def test_set_operations_prec_not_supported(func, grid_size):
+    if "grid_size" not in inspect.signature(func).parameters:
```

Comment:
I would maybe hardcode `if func is shapely.disjoint_subset_union`, to avoid that this might accidentally skip tests if something went wrong in this check